spacy/cli/evaluate.py

```diff
@@ -60,6 +60,7 @@ def evaluate(
     displacy_path: Optional[Path] = None,
     displacy_limit: int = 25,
     silent: bool = True,
+    span_key="sc",
```

Comment:
Let's call this `spans_key` to keep it consistent with the `spancat` config? (Hopefully a future version will parse all the `spans_` scores in some more general purpose way, but we don't want the v3.1 release to wait on this right now.)

Reply:
Ohh, yeah. The consistency helps. Thanks for mentioning this. Have made the change now